src/main/resources/data-repository-openapi.yaml

```diff
@@ -143,6 +163,49 @@ paths:
         403:
           description: No permission to create studies
 
+  '/api/custody/v1/study/{id}':
+    get:  ## retrieve
+        description: Retrieve a study by id
+        operationId: retrieveStudy
+        tags:
+        - model
+        parameters:
+        - in: body
+          name: study
+          description: Id of study to retrieve
+          schema:
+            $ref: '#/definitions/UniqueIdModel'
+        responses:
+          200:
+            description: Created study
+            schema:
+              $ref: '#/definitions/StudyModel'
+          400:
+            description: Bad request - duplicate name, invalid name, badly formed schema
+          403:
+            description: No permission to see study
+
+    delete: ## delete
+      description: Delete a study by id
+      operationId: deleteStudy
+      tags:
+      - custody
+      parameters:
+      - in: body
+        name: study
+        description: Id of study to delete
+        schema:
+          $ref: '#/definitions/UniqueIdModel'
+      responses:
+        204:
+          description: Study deleted successfully (or already did not exist)
+        400:
+          description: Bad request - invalid id, badly formed ## assuming invalid means cannot exist--not _does not_ exist
+        403:
```

Comment:
I think we need a 404 here also. I think this result is missing from all of the {id} based lookups. I'm not going to comment each one.